packages/controls/src/widget_image.ts

```diff
@@ -43,9 +44,13 @@ class ImageView extends DOMWidgetView {
          * Called when the model is changed.  The model may have been
          * changed by another view or by a state update from the back-end.
          */
-        var image_src = 'data:image/' + this.model.get('format') + ';base64,' + this.model.get('_b64value');
-        this.el.setAttribute('src', image_src);
-
+        var blob = new Blob([this.model.get('value')], {type: `image/${this.model.get('format')}`});
+        var url = URL.createObjectURL(blob);
+        var oldurl = this.el.src;
+        this.el.src = url;
```

Comment:
I think the createObjectURL is deprecated (firefox gives you a warning on that).
https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL
The proper way would be to use `this.el.srcObject = blob` I think. We may want to polyfill this with https://github.com/webrtc/adapter , which is what I do in https://github.com/maartenbreddels/ipywebrtc